networkx/algorithms/assortativity/correlation.py

```diff
@@ -134,6 +134,13 @@ def degree_pearson_correlation_coefficient(G, x="out", y="in", weight=None, node
     r : float
        Assortativity of graph by degree.
 
+    Raises
+    ------
+    NetworkXError
+        If the graph 'G' is empty.
+    ValueError
```

Comment:
Yes, I agree with you as it will appropriately point that error is related to graph properties and not the input values. Thankyou for the review! I'll make this change.